pangeo_forge_orchestrator/model_builders.py

```diff
@@ -122,10 +122,10 @@ def make_table_cls(self) -> SQLModel:
         cls_name = self.make_cls_name(self.base, "")
         attrs = dict(id=Field(default=None, primary_key=True))
         annotations = dict(id=Union[int, None])
-        attrs.update(dict(__annotations__=annotations))
+        attrs |= dict(__annotations__=annotations)
```

Comment:
What does this operator do? Is it the same as update? It's new to me but happy to learn something new! 😄

Reply:
yes, this is a shorthand for dict update: https://peps.python.org/pep-0584/... i believe it's available for py39+.... happy to revert to the more explicit `dict.update`